src/content/docs/kubernetes-pixie/kubernetes-integration/troubleshooting/helm-configurations-not-applied.mdx

```diff
@@ -9,11 +9,11 @@ tags:
 
 ## Problem [#problem]
 
-You have completed the [installation procedure](/docs/kubernetes-monitoring-integration#install) for New Relic's Kubernetes integration with helm for the `nri-bundle`, but the values in your `values.yaml` file are not being respected.
+You have completed the [installation procedure](/docs/kubernetes-monitoring-integration#install) for New Relic's Kubernetes integration with Helm for the `nri-bundle`, but the values in your `values.yaml` file are not being respected. Some [global values](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle#values) have been found to not be respected by our Helm templates. 
```

Comment:
@reese-lee could we combine your comment into the original sentence like this? Is this still accurate?

```
You have completed the [installation procedure](/docs/kubernetes-monitoring-integration#install) for New Relic's Kubernetes integration with Helm for the `nri-bundle`, but our Helm templates are not respecting some  [global values](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle#values) in your `values.yaml`. 
```

Reply:
Yes, and thank you for condensing this further!

Reply:
Just committed your suggestion.

Reply:
Thanks!